src/post/post.service.ts

```diff
@@ -582,23 +591,23 @@ export class PostService {
 
         // If the provided handle is invalid, return early
         if (!isHandle(handle)) {
-            throw new Error('Invalid handle');
+            return error('invalid-handle');
```

Comment:
We already validate `handle` in the HTTP handler - why do we need to validate it again? Can we pass the `apId` directly to this method instead?